src/scenes/AcceleratorRouter/ParticipantProjects/SingleView.tsx

```diff
@@ -212,6 +217,24 @@ const SingleView = () => {
                 userLabel={strings.BY}
               />
             </Grid>
+            <Grid container>
+              <Grid item xs={12} margin={`0 ${theme.spacing(2)}`}>
+                <Typography fontSize='20px' fontWeight={600} lineHeight='28px'>
+                  {strings.CARBON}
+                </Typography>
+              </Grid>
+              <ProjectFieldDisplay
+                label={strings.MIN_MAX_CARBON_ACCUMULATION}
+                value={
+                  participantProject?.minCarbonAccumulation && participantProject?.maxCarbonAccumulation
+                    ? `${participantProject.minCarbonAccumulation}-${participantProject.maxCarbonAccumulation}`
+                    : false
+                }
+              />
+              <ProjectFieldDisplay label={strings.CARBON_CAPACITY_TC02_HA} value='770' />
+              <ProjectFieldDisplay label={strings.ANNUAL_CARBON_T} value='82,566' />
+              <ProjectFieldDisplay label={strings.TOTAL_CARBON_T} value='3,302,640' />
```

Comment:
I wasn't able to find new carbon properties for these fields in the API schema with a quick search, but these hardcoded values need to be replaced once the properties are available.

Reply:
Good catch. I think they need to be added to existing Accelerator project details as well. Let me look into new columns in the BE.

Reply:
https://github.com/terraware/terraware-server/pull/2212

Reply:
Ready now!